int16 wheel1
int16 wheel2
int16 wheel3
int16 wheel4
